package.json

```diff
@@ -276,11 +276,6 @@
                 "title": "%azureFunctions.restartFunctionApp%",
                 "category": "Azure Functions"
             },
-            {
-                "command": "azureFunctions.setAzureWebJobsStorage",
```

Comment:
> Also setAzureWebJobsStorage is never actually invoked anywhere, so I went ahead and deleted that.

I use it a decent amount please don't remove it 🥺 If you clone a function app repo or just accidentally delete your "local.settings.json" it's a fast way to get the file back

Reply:
Hahaha okay. I figured the only entry point was through the command palette, so who could even be using it...? But I guess you are 1 of 85 people per month.

![image](https://github.com/microsoft/vscode-azurefunctions/assets/5290572/2488dc4e-37d2-43e9-9518-7ecce2b2cee8)